test/helpers.ts

```diff
@@ -8,7 +8,8 @@ export function execComposeUp() {
   execSync('cd test && docker-compose up -d && sleep 5');
 }
 
-export function execComposeDown() {
+export async function execComposeDown(input: { modules?: string[], region?: string }) {
```

Comment:
curious, why an object and not just two optional parameters?

Reply:
I was having an issue with the second argument, but I think I know how to solve it now